README.md

```diff
@@ -389,7 +389,9 @@ class App extends React.Component {
 
 Inside `<InjectedCheckoutForm />`, `this.props.stripe` will either be `null` or
 a Stripe instance. React will re-render `<InjectedCheckoutForm>` when
-`this.props.stripe` changes from `null` to a Stripe instance.
+`this.props.stripe` changes from `null` to a Stripe instance. Note that when
+providing Stripe.js asynchronously, you can still wrap a component returned by the
+[`injectStripe` HOC](#injectstripe-hoc).
```

Comment:
> you can still wrap a component returned by the [`injectStripe` HOC](#injectstripe-hoc)

This sentence is a little confusing, since it's not clear from the preceding context why this says "you can still". What is the alternative? Why would a reader assume you can't use `injectStripe` when loading Stripe asynchronously?

Reply:
Reading #160 makes this clearer. Perhaps rephrase as something like: "When loading Stripe.js asynchronously, the `stripe` prop provided by `injectStripe` will be initially null, and will become the Stripe instance once the instance is passed in."

Reply:
Ah yeah, really like your suggested sentence there. It flows more naturally and gets the point across.